modules/deployments-ab-testing-lb.adoc

```diff
@@ -113,23 +113,36 @@ spec:
 ----
 
 [id="deployments-ab-testing-lb-web_{context}"]
-== Managing weights using the web console
+== Managing weights of an existing route using the web console
 
 .Procedure
 
-. Navigate to the Route details page (Applications/Routes).
+. Navigate to the *Networking* -> *Routes* page.
 
-. Select *Edit* from the Actions menu.
+. Click the Actions menu {kebab} next to the Route you want to edit and select *Edit Route*.
 
-. Check *Split traffic across multiple services*.
+. Edit the YAML file. Update the `weight` to be an integer between `0` and `256` that specifies the  target's relative weight against other target reference objects. The value `0` suppresses requests to this back-end. The default is `100`. Run `oc explain routes.spec.alternateBackends` for more information about the options.
 
-. The *Service Weights* slider sets the percentage of traffic sent to each service.
-+
-For traffic split between more than two services, the relative weights are
-specified by integers between 0 and 256 for each service.
-+
-Traffic weightings are shown on the *Overview* in the expanded rows of the
-applications between which traffic is split.
+. Click *Save*.
+
+[id="deployments-ab-testing-lb-web-new-route_{context}"]
+== Managing weights of an new route using the web console
+
+. Navigate to the *Networking* -> *Routes* page.
+
+. Click the *Create Route* button.
```

Comment:
Unless this is consistent with other docs in this book, I would shorten as we do with "Save" and write `Click *Create Route*.`